eng/doc/fips/README.md

```diff
@@ -42,7 +42,7 @@ The Microsoft Go fork provides several ways to configure the crypto backend and
   - [`GOEXPERIMENT` `allowcryptofallback`](#build-option-to-use-go-crypto-if-the-backend-compatibility-check-fails)
```

Comment:
Line above this one (too far from a diff to comment directly) mentions `requirefips` and should maybe also mention `GOFIPS140=latest`.